### ISSUE TITLE
learn vue.js basics for frontend

### ISSUE THREAD

**Issue by KayaOndul** [KayaOndul/SWE-573#18]
Vuejs will be learned to implement frontend of the application

**Comment by KayaOndul** [comment]
During Saturday and Sunday, I went through the halfway of vuejs course. While completing exercise tasks, i also made a research about the WebStorm which is the IDE for Javascript ES6. However, Intellij IDEA is sufficient for this task. To keep everything in one place i will continue developing with IJ IDEA.

I plan to finish the course of  Vue.js until Wednesday. Meantime I will try to implement user login and registration pages for exercise.

Total time spent is 14 hours

**Comment by KayaOndul** [comment]
Today, I continued to work on Vue. As I progressed with the Udemy course, I decided that learning by doing is more effective. I completed Gwen Faraday's course on youtube which is 3 hours long. During the lesson, I also replicated the code parts, sometimes stopped the video and tried to code by myself.

End of the day I got some confidence in Vue, Vuex, Vue Router. I think I am ready to implement the frontend of my app. 

I plan to learn more advanced features if they become a necessity.

Right now I am comfortable with it, hence I close this issue.
